CONTRIBUTING.md

```diff
@@ -187,7 +187,7 @@ Go to the [generator-jhipster project](https://github.com/jhipster/generator-jhi
 
 ### Set NPM/YARN to use the cloned project
 
-In your cloned `generator-jhipster` project, type `npm link` or `yarn link` depending on the package manager you use.
+In your cloned `generator-jhipster` project, type `npm install && npm link` or `yarn && yarn link` depending on the package manager you use.
```

Comment:
actually the link command used to do install as well, doesnt it do anymore?

Reply:
I just checked and it does install for `npm link` but not for `yarn link`.